instagram-app/src/components/Login/Login.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+
+class Login extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            username: '',
+            password: ''
+        };
+    }
+
+    handleLogin = event => {
+        this.setState({ [event.target.name]: event.target.value });
+    };
+
+    handlebutton = event => {
+        let user = this.state.username;
+        localStorage.setItem('user', user);
+
+    };
+
+    render() {
+        return (
+            <form className='login'>
+                <input type='text' placeholder='username' name = 'username' value={this.state.username} onChange={this.handleLogin} />
+                <input type='text' placeholder='password'  name = 'password'value={this.state.password} onChange={this.handleLogin} />
+                <button onClick={this.handlebutton}>Log In</button>
```

Comment:
Good use of the `button` element to trigger a refresh with an `onSubmit` event - keep in mind that clicking on a `button` element will trigger an `onSubmit` event unless its `type` attribute is set to `"button"`.